fractal-puppy/Engine
# Bug/enemy bone drop bug

The bone drop was closely related to a modification done for the boss fight, so I had to modify stuff for the boss too.
To test:
- check that now basic enemies and stuff drop bones properly.
- check that boss fight is still working fine.
